### ISSUE TITLE
Hardware resource requirements for UMNT? 

### ISSUE THREAD

**Issue by sebastian-nehrdich** [facebookresearch/XLM#76]
Hello!
Thank you for the amazing work! I am eager to try this on my own datasets.
I have 200MB of Tibetan and about 100MB of Sanskrit data (each monolingual) which I would like to use as basis for unsupervised machine tanslation. What hardware requirements are necessary to train this? Is there any hope to get this done on a single Maxwell Titan X or should I go straight for the cloud TPU? Also once the model is trained, is inference possible on reasonable time on a single GPU?
Any hint is highly appreciated!

**Comment by glample** [comment]
Hi,

When a model is trained, 1 GPU is definitely enough for inference. To train it, it is better to have multiple GPUs though, unless your training data is very small. In practice we usually observe better performance with several GPUs, as the batch size is much larger. Or you could try to accumulate the gradients to simulate the same effect on a single GPU, but this would require some modifications in the code.
